app/models/manageiq/providers/vmware/infra_manager.rb

```diff
@@ -169,8 +169,7 @@ def validate_remote_console_webmks_support
     end
 
     def self.use_vim_broker?
```

Comment:
should this method be removed altogether?

Reply:
:+1: yes, was going to leave this at first so that this could be merged independently from core, but there are other inter-dependencies now anyway